Classes/Form/Field/Text.php

```diff
@@ -50,6 +60,10 @@ public function buildConfiguration() {
 		} else {
 			$configuration['defaultExtras'] = $defaultExtras;
 		}
+		if (FALSE === empty($this->getRenderType())) {
```

Comment:
This one I think causes an E_NOTICE; you have to assign an actual variable with $renderType then check that with empty().